content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
 ---
-description: ' Use the Viewport Interaction Model to interact with entities in the Open 3D Engine viewport.
-  Learn about the differences between the new Viewport Interaction Model and the old interaction model. '
-linktitle: Viewport Interaction Model
-title: Working with the Viewport Interaction Model
+linkTitle: 3D Viewport
+title: Working with the 3D Viewport
+description: Use the 3D Viewport to interact with entities in Open 3D Engine (O3DE).
+weight: 400
 ---
 
-The Viewport Interaction Model (VIM) allows users to interact with entities in the **Open 3D Engine (O3DE)** viewport. This feature provides selection, translation, rotation, and scale. With the Viewport Interaction Model, you can create custom reference spaces to transform your entities. With custom reference spaces, you can move your selected entities in relation to the reference that you specify. The Viewport Interaction Model also unifies entity and component manipulators so that you can interact with entities and components in the same way.
+In **Open 3D Engine** (O3DE), you can interact directly with entities using the 3D Viewport.  The 3D Viewport provides several features to complete common tasks such as editing the location, rotation, and scale of entities, hierarchies of entities, and their components.  [UI widgets](#ui-widgets), [Manipulators](#manipulators), [Hotkeys](#hotkeys), and visual feedback mechanisms simplify the process of editing and arranging entities. With [Reference spaces](#reference-spaces), you can quickly select if entities are transformed in world space, local space, in relation to a parent entity, or custom locations.
 
-The Viewport Interaction Model provides the following benefits:
+![Screenshot of 3D Viewport with UI widgets, entity, and a manipulator visible.](/images/user-guide/editor/3d-viewport.png)
 
-+ Improves the ability to select your preferred entity, especially for nested hierarchies or levels that have many entities.
-+ Adds keyboard and mouse actions that you can apply to your selected entities. You can freely switch between local, parent, and world space to change your selected entities.
-+ Establishes an editing pattern that matches that of Component Mode.
+## UI widgets
```

Comment:
suggestion: I think referring to these as 'Viewport UI Widgets' would be clearer (to distinguish from other UI widgets across the editor in general).